deploy/backplane/lpsre/rhoc/01-rhoc-lpsre-admins-cluster.ClusterRole.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-lpsre-rhoc-admins-project
+rules:
+ # LP SRE can run commands in pods
+- apiGroups:
+  - ""
+  resources:
+  - pods/exec
+  verbs:
+  - create
+# LP SRE can portforward pods
+- apiGroups:
+  - ""
+  resources:
+  - pods/portforward
+  verbs:
+  - create
+# LP SRE can delete jobs and builds
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  - cronjobs
+  verbs:
+  - delete
+  - deletecollection
+- apiGroups:
+  - build.openshift.io
+  resources:
+  - builds
+  verbs:
+  - delete
+  - deletecollection
+# LP SRE can use oc debug
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/attach
+  verbs:
+  - create
+# LP SRE can get pod logs
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/log
+  verbs:
+  - get
+# LP SRE can delete pods
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - delete
+# LP SRE can interact with secrets, configmaps and PVC's
+- apiGroups:
+  - ""
+  resources:
+  - secrets
```

Comment:
What is the use case for accessing secrets?  What secrets exist in the namespace?  This is not something normally granted for backplane users.

Reply:
@jewzaam https://github.com/openshift/managed-cluster-config/pull/1397 Here is the PR that originally was made for adding the RHOC role. I have not been involved in the RHOC process to be able to answer this appropriately. I am just copying over what was approved/exists in the repo currently into this new role which is a merge of CSSRE and MTSRE access/roles. It looks like based on the PR and feedback though it was cleared.

Reply:
Fair enough, I missed that in the prior review then.  Can you find out why secret is needed?  If it's justified, fine, else it can be dropped.

Reply:
I agree we shouldn't be able to read secrets by default.  If we have a true reason we need to do so, we can escalate and provide appropriate justification on the Compliance JIRA.

Reply:
```suggestion
```